python/test/test_meas.py

```diff
@@ -252,3 +250,59 @@ def test_randomized_response_bitvec():
     # epsilon is 2 * m * ln((2 - f) / f)
     # where m = 4 and f = .95
     assert m_rr.map(1) == 0.8006676684558611
+
+def test_laplace_threshold_int():
+    domain = dp.map_domain(dp.atom_domain(T=str), dp.atom_domain(T=int))
+    metric = dp.l_01I(dp.absolute_distance(T=int))
+    meas = dp.m.make_laplace_threshold(domain, metric, scale=2., threshold=28)
+    release = meas({str(i): i * 10 for i in range(10)})
+    # 0 + noise is likely not over 28
+    assert "0" not in release, release
+    # 90 + noise is likely over 28
+    assert "9" in release, release
```

Comment:
```suggestion
    # 0 + noise is likely not over 28
    assert "0" not in release
    # 90 + noise is likely over 28
    assert "9" in release
```
Default pytest error messages are enough:
```
    def test_fake():
>       assert 'a' not in 'abcdefg'
E       AssertionError: assert 'a' not in 'abcdefg'
E
E         'a' is contained here:
E           abcdefg
```